QUEST_LV_0100.tsv

```diff
@@ -5889,7 +5889,7 @@ QUEST_LV_0100_20150317_005888	{memo X}You went to Crystal Mines to find the evid
 QUEST_LV_0100_20150317_005889	You followed the revelation of the Goddess that the bishop saw in this dreams and obtained the Slate. Talk to Knight Commander Uska about the slate.
 QUEST_LV_0100_20150317_005890	Ask the Bokor Master for interpretation of the revelation.
 QUEST_LV_0100_20150317_005891	{memo X}Ask Bokor Mama about this slate. She lives in the left end of Klaipeda residential area.
-QUEST_LV_0100_20150317_005892	The slate found in Crystal Mines is the revelation and Goddess Laima herself. It contains a message about finding the next revelation in holy church. Return to Uska as the Bokor Master says.
+QUEST_LV_0100_20150317_005892	The slate found in Crystal Mines is the revelation and Goddess Laima herself. It contains a message about finding the next revelation in holy church. Return to Sir Uska as the Bokor Master says.
```

Comment:
*the* Crystal Mine 
Holy church -> chapel (can't check the original on my mobile, but the next revelation is in a map named Tenet Chapel)